signup-service/src/test/java/io/mosip/signup/services/RegistrationServiceTest.java

```diff
@@ -1137,6 +1431,68 @@ public void register_thenAddIdentityEndpointResponseErrors_throwException() thro
         }
     }
 
+    @Test
+    public void register_thenAddIdentityEndpointResponseNullErrors_throwException() throws Exception{
```

Comment:
On null errors why should we throw an exception?

Reply:
this test case for when register response and error are null